src/components/plots/ContinuousEmbeddingPlot.jsx

```diff
@@ -36,7 +36,7 @@ const ContinuousEmbeddingPlot = (props) => {
   const [plotSpec, setPlotSpec] = useState({});
 
   useEffect(() => {
-    if (cellSets.loading && !cellSets.error) {
+    if (!cellSets.error) {
```

Comment:
some questions:
1. when we load a new embeddings method the old cellsets are discarded so we need to reload them again, is that right?
2. this useEffect() depending on `experimentId` makes sense? how can the experimentId change while staying on the same plot? afaik the only way for the experimentId to change is to navigate to another experiment (which will be a new page) and hence the cellsets can be loaded as part of the initial load. Is that true?

Reply:
> 1. when we load a new embeddings method the old cellsets are discarded so we need to reload them again, is that right?

No, the cell sets are always the same (the ones stored in the cell-sets-* bucket). They just need to be downloaded if they haven't been yet.

What's happening here is it's calling `loadCellSets` but after the cellSets are loaded the first time nothing will happen anymore. Actually, having `loadCellSets` in this `useEffect` is probably a bit confusing, it would be more clear for it to be in a `useEffect` of its own without any dependency.